distribution/lib/Standard/Base/0.0.0-dev/src/Meta/Enso_Project.enso

```diff
@@ -91,6 +91,11 @@ type Project_Description
     namespace : Text
     namespace self = self.ns
 
+    ## PRIVATE
+       Returns the path of the project root.
+    root_path : Text
+    root_path self = self.root.path
```

Comment:
It feels a bit redundant, since we can just call `invokeMethod` twice on Java side, right?

Reply:
I did this because `invokeMember` didn't work here; I get:

```
Exception occurred in target VM: Unsupported operation Value.invoke(path, Object...) for '(File /Users/gmt/dev/enso/enso/test/Table_Tests)'(language: Java, type: org.graalvm.polyglot.Value). You can ensure that the operation is supported using Value.canInvoke(String).
```

Perhaps because it's a builtin Enso type?

```
root.hasMember("path")
false
```

Reply:
Oh, I see. Well in such case such workaround seems justified.

Reply:
I've just had similar experience with `Ref` - its methods aren't accessible via `invokeMember` - I guess that's a bug and should be fixed. All builtins shall have their methods available via `org.graalvm.polyglot.Value` API.

Reply:
- reported as #11589